test/test.js

```diff
@@ -156,3 +156,21 @@ test('missingHighwaysUS', function(t) {
     t.end();
   });
 });
+
+
+test('crossingHighways', function(t) {
+  t.plan(1);
+  logInterceptor();
+  processors.crossingHighways(opts, mbtiles, function() {
+    var logs = logInterceptor.end();
+    for (var i = 0; i < logs.length; i++) {
+      var geoJSON = JSON.parse(logs[i]);
+      t.comment('Pass: ' + (i + 1));
+      if (geoJSON.features.length > 0) {
+        t.equal(geoJSON.features[0].properties._osmlint, 'crossinghighways', 'Should be crossinghighways');
+        //geoJSON.features[0].geometry.type = LineString, Point, MultiPOint
```

Comment:
@geohacker  when we have the more than tree types of output, how should we do the testing?  in this case the output of Crossinghighways validator is  `LineString, Point, MultiPoint`.   what do you reckon?